lib/echo/base.rb

```diff
@@ -17,7 +17,7 @@ def build_connection
         conn.use :instrumentation
 
         # Set timeout to 300s to match nginx timeout
-        conn.options[:timeout] = NGINX_TIMEOUT
+        conn.options[:timeout] = NGINX_TIMEOUT - 10
```

Comment:
in the order tracking timeout catching we start with `echo_client.timeout - 30`, so should that be adjusted slightly if we are changing this?

Reply:
Well I was thinking that code dealing with the -30 was a slightly more computation intensive process, why I did the -30.   The -10 here are defaults.